app/views/ingredients/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class='container wrapper'>
+  <div class='row'>
+    <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
+      <%= form_for @ingredient, url: {action: 'create'} do |i| %>
+      <form>
```

Comment:
- `form_for` is already creating `form` element for you ;).
- Please nest content of `form_for` block.